pkg/cloud/ibmcloud/actuators/machine/actuator.go

```diff
@@ -63,6 +69,8 @@ func NewActuator(params ibmcloud.ActuatorParams) (*IbmCloudClient, error) {
 func (ic *IbmCloudClient) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	klog.Infof("Creating machine %v for cluster %v.", machine.Name, cluster.Name)
 
+	ic.updatePhase(ctx, machine, MachinePending)
```

Comment:
why we have pending status? if you see line 90 this means for already created instance you create it again 
you will mark as pending? 

I think just creating is enough ? if you agree I can create a PR to update it @xunpan

Reply:
Please refer to #16 for more discussion